config.mk

```diff
@@ -218,8 +218,8 @@ YUM_DOWNLOAD_SRC?=
 
 MIRROR_CENTOS?=http://mirrors-local-msk.msk.mirantis.net/centos-$(PRODUCT_VERSION)/$(CENTOS_RELEASE)
 MIRROR_CENTOS_KERNEL?=http://mirror.centos.org/centos-6/6.6/
```

Comment:
You're right, I used to use shell env vars to override MIRROR_CENTOS* and supply EXTRA_RPM_REPOS. I'll update this.